server/db/index.js

```diff
@@ -0,0 +1,13 @@
+const { Client } = require('pg');
+
+const client = new Client({
+  host: 'localhost',
```

Comment:
You should pull these variables from dotenv instead of hardcoding

Reply:
changed this! thank you!